### PR TITLE
fix(#1368): long list of entities in Token Classifier

### LINKED ISSUE #1368: Odd behavior for some labels in TokenClassificationRecord
**Describe the bug**

For a TokenClassificationRecord, some of the predictions don't show up correctly. As shown in the image below, the word 'Snoring' isn't underlined like the word 'Fatigue' is. And when I hover over 'Snoring' the expected pop up is displayed without color. For context, we have many labels (~100).

**To Reproduce**
1. Upload many TokenClassifcationRecords with many labels and predictions.

**Expected behavior**

All tokens that correspond to predictions are displayed in the same manner. Underlined with a color, etc...

**Screenshots**

Screen Shot 2022-03-31 at 2 22 00 PM

**Environment (please complete the following information):**
 - OS [e.g. iOS]: Linux / Mac
 - Browser [e.g. chrome, safari]: Chrome
 - Rubrix Version [e.g. 0.10.0]: 0.13.0
 - ElasticSearch Version [e.g. 7.10.2]:
 - Docker Image (optional) [e.g. rubrix:v0.10.0]:

### LINKED ISSUE #1393: When TokenClassificationRecrod has a lot of labels, if I expand the labels to view them all, the screen doesnt allow me to scroll down to view all the labels
When TokenClassificationRecrod has a lot of labels, if I expand the labels to view them all, the screen doesnt allow me to scroll down to view all the labels. When this happens there is no way to view the records either since the screen is essentiallly stuck.

Screen Shot 2022-04-07 at 9 36 33 AM

### PR DESCRIPTION
**Pull request by leiyre**
fixes #1368
fixes #1393

This PR changes the number of color entities from 50 to infinity and allows scroll in entities header

### CODE REVIEW

File: frontend/models/TokenClassification.js [hunk 1]
```diff
@@ -102,8 +102,9 @@ class TokenClassificationDataset extends ObservationDataset {
       ),
     ];
     return names.map((name, index) => {
+      const colorLimit = 50;
```
Comment (frascuchon) [inline frontend/models/TokenClassification.js]: This magic number (50) should be used where the color palette is defined.

**Review by frascuchon — CHANGES_REQUESTED** [review]

**Comment by frascuchon** [comment]
Also, with these changes, maybe we could consider decreasing the number of available colors for entity tags?

cc @leiyre @Amelie-V @dvsrepo

**Comment by leiyre** [comment]
> Also, with these changes, maybe we could consider decreasing the number of available colors for entity tags?
> 
> cc @leiyre @Amelie-V @dvsrepo

Yes, we can experiment with more contrasting colors by limiting the number or we can also pass a list of custom colors.

File: frontend/specs/components/token-classification/results/__snapshots__/EntitiesSelector.spec.js.snap [hunk 2]
```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EntitiesSelector renders properly 1`] = `
-<span class="entities__selector__container"><div class="entities__selector"><ul class="entities__selector__options"><li class="entities__selector__option suggestion color_14 active"><span>Arreglo</span> <span class="entity__sort-code">[space]</span></li>
-<li tabindex="0" class="entities__selector__option color_14"><span>Arreglo</span> <span class="entity__sort-code">[1]</span></li>
-<li tabindex="0" class="entities__selector__option color_1"><span>CARDINAL</span> <span class="entity__sort-code">[2]</span></li>
+<span class="entities__selector__container"><div class="entities__selector"><ul class="entities__selector__options"><li class="entities__selector__option suggestion color_NaN active"><span>Arreglo</span> <span class="entity__sort-code">[space]</span></li>
+<li tabindex="0" class="entities__selector__option color_NaN"><span>Arreglo</span> <span class="entity__sort-code">[1]</span></li>
+<li tabindex="0" class="entities__selector__option color_NaN"><span>CARDINAL</span> <span class="entity__sort-code">[2]</span></li>
```
Comment (frascuchon) [inline frontend/specs/components/token-classification/results/__snapshots__/EntitiesSelector.spec.js.snap]: `color_NaN` are expected results? This snapshot shouldn't have changed

**Comment by frascuchon** [comment]
After deploy changes the webapp  is not accessible and I see the following error:
Screenshot 2022-04-07 at 17 25 43

**Review by frascuchon — APPROVED** [review]

**Pull request merged**